sfall/Modules/BarBoxes.cpp

```diff
@@ -34,6 +38,17 @@ struct sBox {
 static sBox boxes[10];
 static DWORD boxesEnabled[5];
 
+#define tSize    28
```

Comment:
why changing tSize to 28? Text length over 16 characters doesn't fit the box well.

Reply:
characters have a different size, can fit up to 18 characters. made the buffer size 20 with a margin.